contrib/debian/fwupd.postinst

```diff
@@ -49,9 +49,6 @@ for dir in /var/cache/fwupdate /var/lib/fwupdate; do
 	fi
 done
 
-#create a user for fwupd-refresh.service/fwupd-refresh.timer
```

Comment:
Presumably need to drop the postrm stuff too.

Reply:
Oh and this only works if the meson command in debian/rules sets the systemd_unit_user (don't remember if it does).

Reply:
Yup, we're using `-Dsystemd_unit_user=fwupd-refresh` -- but I'm a bit confused what needs to change in fwupd.postrm -- that only seems to be dealing with stuff in /etc.

Reply:
Oh ha; I looked and we're missing the cleanup step for the user anyway in `postrm`.
This will need some upgrade testing on the packaging after this merges because having the older user made by postinst and this new user made by systemd there _could_ be some conflicts.

Reply:
Do you mind looking at the debian bits please; i have no idea what I'm doing there.